BuildResidentialScheduleFile/resources/schedules.rb

```diff
@@ -244,17 +236,17 @@ def create_stochastic_schedules(args:)
       end
     end
 
-    sink_duration_probs = schedule_config['sink']['duration_probability']
-    events_per_cluster_probs = schedule_config['sink']['events_per_cluster_probs']
-    hourly_onset_prob = schedule_config['sink']['hourly_onset_prob']
+    sink_duration_probs = Schedule.validate_values(Constants.SinkDurationProbability, 9, 'sink_duration_probability')
+    events_per_cluster_probs = Schedule.validate_values(Constants.SinkEventsPerClusterProbs, 15, 'sink_events_per_cluster_probs')
+    hourly_onset_prob = Schedule.validate_values(Constants.SinkHourlyOnsetProb, 24, 'sink_hourly_onset_prob')
     # Lookup avg_clusters_per_occ from json
```

Comment:
Comment still refers to json. Remove/update and look for any other old references to the json config file.